i3bar/src/xcb.c

```diff
@@ -1889,6 +1847,18 @@ void reconfig_windows(bool redraw_bars) {
             }
         }
     }
+
+    /* Finally, check if we want to initialize the tray or destroy the selection
+     * window. The result of get_tray_output() is cached. */
+    if ((output_for_tray = get_tray_output())) {
```

Comment:
I would slightly change this so that it's clear this isn't a case of an accidental assignment over comparison and makes it just a bit easier to read.

```suggestion
    output_for_tray = get_tray_output();
    if (output_for_tray) {
```